config/taxonomy/module.values.schema.json

```diff
@@ -9,33 +9,27 @@
             "description": "Type of modules supported by the M4D manager.",
             "enum": ["copy", "read", "write"]
         },
-        "transformation_name": {
-            "description": "Transformation that can be performed on data.",
+        "protocol": {
             "type": "string",
-            "enum": ["mask", "redact", "encrypt"]
-        },
-        "data_level_name": {
-            "type":"string",
-            "enum":["UNKNOWN", "DATASET", "COLUMN", "ROW", "CELL"]
+            "description": "The mechanism via which the data will be accessed.",
+            "enum": ["s3", "kafka", "db2-jdbc", "m4d-arrow-flight"]
```

Comment:
How is this related to this PR? Will conflict with the work @eletonia is doing.

Reply:
Removed protocol to avoid conflict with Tonia.